common/model-views.cpp

```diff
@@ -3743,30 +3743,46 @@ namespace rs2
                                                                        true );
             }
 
-            if( is_upgradeable( fw, recommended ) )
+            auto dev_name = get_device_name(dev);
+            if( is_upgradeable( fw, recommended_fw_ver) )
             {
-                auto dev_name = get_device_name( dev );
                 std::stringstream msg;
                 msg << dev_name.first << " (S/N " << dev_name.second << ")\n"
                     << "Current Version: " << fw << "\n";
 
                 if( is_rc )
-                    msg << "Release Candidate: " << recommended << " Pre-Release";
+                    msg << "Release Candidate: " << recommended_fw_ver << " Pre-Release";
                 else
-                    msg << "Recommended Version: " << recommended;
+                    msg << "Recommended Version: " << recommended_fw_ver;
 
                 auto n = std::make_shared< fw_update_notification_model >( msg.str(),
                                                                            manager,
                                                                            false );
-                n->delay_id = "dfu." + dev_name.second;
+                // The FW update delay ID include the dismissed recommended version and the device serial number
+                // This way a newer FW recommended version will not be dismissed 
+                n->delay_id = "fw_update_alert." + recommended_fw_ver + "." + dev_name.second;
                 n->enable_complex_dismiss = true;
+
+                if( reset_delay ) n->reset_delay();
+
                 if( ! n->is_delayed() )
                 {
                     not_model->add_notification( n );
                     related_notifications.push_back( n );
+                    return true;
                 }
             }
+            else
+            {
+                std::stringstream msg;
+                msg << "Bundled FW is up to date for: " << dev_name.first << " (S/N " << dev_name.second << ")\n"
```

Comment:
You mean that the current version is up-to-date with the bundled/recommended, right?

Reply:
Yes, I changed it to "Current FW >= Bundled FW for: ..."